index.html

```diff
@@ -122,6 +122,36 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+
+        <div class="card">
+          <p class="name">Prakhar</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/prakhar-agra-10" target="_blank">prakhar-agra-10</a>
```

Comment:
```suggestion
            <a href="https://github.com/prakharagrawal10" target="_blank">prakharagrawal10</a>
```
this is your correct github url